be/src/vec/exec/format/parquet/parquet_column_convert.h

```diff
@@ -671,8 +683,16 @@ inline Status get_converter(tparquet::Type::type parquet_physical_type, Primitiv
         break;
     case TypeIndex::DateTimeV2:
         if (tparquet::Type::INT96 == parquet_physical_type) {
+            // int96 only stores nanoseconds in standard parquet file
+            convert_params->reset_time_scale_if_missing(9);
             *converter = std::make_unique<Int96toTimestamp>();
         } else if (tparquet::Type::INT64 == parquet_physical_type) {
+            int ts_scale = remove_nullable(dst_data_type)->get_scale();
+            int scale = 6;
+            if (ts_scale <= 3) {
```

Comment:
In the Parquet standard, there are indeed only four precisions: seconds (0), milliseconds (3), microseconds (6), and nanoseconds (9). Does this implementation ignore the situation where the precision is seconds(0). Then, if this rule is universal, you can write the rule to `reset_time_scale_if_missing `.